examples/dev/testManiFactors.jl

```diff
@@ -93,8 +111,8 @@ f1 = addFactor!(fg, [:x0,:x1])
 ## Sphere(n)
 ## ======================================================================================
 
-Base.convert(::Type{<:Tuple}, M::Sphere{2, ℝ}) = (:Euclid, :Euclid, :Euclid)
-Base.convert(::Type{<:Tuple}, ::IIF.InstanceType{Sphere{2, ℝ}})  = (:Euclid, :Euclid, :Euclid)
+Base.convert(::Type{<:Tuple}, M::Sphere{2, ℝ}) = (:Euclid, :Euclid)
+Base.convert(::Type{<:Tuple}, ::IIF.InstanceType{Sphere{2, ℝ}})  = (:Euclid, :Euclid)
 
 @defVariable Sphere2 Sphere(2) [1.0, 0.0, 0.0]
```

Comment:
don't worry about Sphere(2) now if it is not compatible with current KDE.  That will later be replaced by AMP.  See JuliaRobotics/ApproxManifoldProducts.jl#41

Reply:
It's a good way to catch dimension issues without taking the full step to have matrix or `ProductRepr` for a point.

Reply:
But, yes, I changed it to `@test_broken`